tests/testthat/helper-http.R

```diff
@@ -0,0 +1,77 @@
+cache <- new_environment()
+
+httpbin_service <- function() {
+  app <- env_cache(
+    cache,
+    "test_app",
+    webfakes::new_app_process(webfakes::httpbin_app())
```

Comment:
It's installed in CI because we have `any::rcmdcheck` in our workflow, and that package requires `callr`.

https://github.com/rstudio/rsconnect/blob/38c19e93170a56e27137652d36113e580b98d32e/.github/workflows/R-CMD-check.yaml#L49

Reply:
Ah, good catch.